catroid/src/main/java/org/catrobat/catroid/ui/WebViewActivity.java

```diff
@@ -189,8 +168,21 @@ public boolean onKeyDown(int keyCode, KeyEvent event) {
 	}
 
 	private class MyWebViewClient extends WebViewClient {
+
 		@Override
 		public void onPageStarted(WebView view, String urlClient, Bitmap favicon) {
+			if (progressCircle == null) {
+				progressCircle = new ProgressDialog(view.getContext(), R.style.WebViewLoadingCircle);
+				progressCircle.setCancelable(true);
+				progressCircle.setCanceledOnTouchOutside(false);
+				progressCircle.setProgressStyle(android.R.style.Widget_ProgressBar_Small);
+				try {
+					progressCircle.show();
+				} catch (Exception e) {
```

Comment:
Got what you meant, but this code existed from before, should i still change it?